docker-compose.yaml

```
@@ -2,6 +2,18 @@ name: lumigator
 
 services:
 
+  inference-model:
+    build:
+      context: .
+      dockerfile: cache/Dockerfile.model-inference
+    platform: linux/${ARCH}
+    command: /bin/true
+    volumes:
+      - ${HOME}/.cache/huggingface:/home/ray/.cache/huggingface
```

```suggestion
      - ${HF_HOME}:/home/ray/.cache/huggingface
```

We have just fixed the HF cache issue and refactored the code a bit.
The main idea now is that, consistently with [HuggingFace's definition](https://huggingface.co/docs/huggingface_hub/main/en/package_reference/environment_variables#hfhome), we map the HF cache dir in the container to a HF_HOME dir whose default is ${HOME}/.cache/huggingface, but that can be customised if the user is already using a different place for that (see https://github.com/mozilla-ai/lumigator/pull/935).
Sorry for the change under the hood!

---

updated!